travis/run_pylint.py

```diff
@@ -12,13 +12,16 @@
 CLICK_DIR = click.Path(exists=True, dir_okay=True, resolve_path=True)
 
 
-def get_count_fails(linter_stats):
+def get_count_fails(linter_stats, msgs_no_count=None):
     """Verify the dictionary statistics to get number of errors.
     :param linter_stats: Dict of type pylint.lint.Run().linter.stats
+    :param no_count: List of messages that will not add to the failure count.
     :return: Integer with quantity of fails found.
     """
     count = 0
     for msg in linter_stats['by_msg']:
+        if msg in msgs_no_count:
+            continue
         count += linter_stats['by_msg'][msg]
```

Comment:
Think function could be a one liner :-)

    return sum([
        linter_stats['by_msg'][msg] 
        for msg in linter_stats['by_msg'] 
        if msg not in msgs_no_count])